daemon/src/auth.rs

```diff
@@ -14,7 +14,8 @@ use std::str::FromStr;
 /// A request guard that can be included in handler definitions to enforce authentication.
 pub struct Authenticated {}
 
-pub const MAKER_USERNAME: &str = "maker";
+pub const USERNAME: &str = "itchysats";
+pub const TAKER_DEFAULT_PASSWORD: &str = "onbrinkofsecondbailout";
```

Comment:
I'd rather not have a default password in the binary. If we want a default password for umbrel, we can always hardcode that in the docker-compose file. If `--password` is not specified, we can always derive a password from the seed.

What do you think?

Reply:
also: if we would like a default password for the CI, we can always hardcode one in `cargo dev-taker` etc.